overlay/etc/confconsole/services.txt

```diff
@@ -1,5 +1,5 @@
 Store front: https://$ipaddr/
-Admin panel: https://$ipaddr/admin
+Admin panel: https://$ipaddr/turnkey_admin
```

Comment:
Great catch! These details are easy to forget, so nice work man! :)

Although I do note that the Webshell line is still there. We don't provide webshell anymore in v18.0, so that can be removed (and the port 12320 removed from the Makefile too).